joomla/joomla-cms
# [com_languages] Solve PHP warnings if language metafile does not exist

Pull Request for new Issue.
#### Summary of Changes

This PR is to remove PHP warnings when a language XML meta file is not found.
##### Before patch

![image](https://cloud.githubusercontent.com/assets/9630530/14249857/b044f72c-fa73-11e5-8c66-09e0002d4a40.png)
##### After patch

![image](https://cloud.githubusercontent.com/assets/9630530/14249868/b8e30824-fa73-11e5-8326-0910886acb08.png)
#### Testing Instructions
1. Use 3.5.0 or latest staging
2. Install a second language (ex: fr-FR)
3. Go to `/language` folder and rename the `fr-FR` directory to anything you like.
4. In Joomla admin, go to `Extensions -> Language(s) -> Installed" you will get the PHP warning.
5. Apply patch.
6. Repeat step 4, you will get a joomla warning message.
   (you can now rename the folder back to the is original name)

Note: the same check if the xml file exists is made in `JInstaller::parseXMLInstallFile()`
